src/main/scala/com/gu/util/TargetHost.scala

```diff
@@ -0,0 +1,64 @@
+package com.gu.util
+
+import com.typesafe.config.ConfigFactory
+import org.slf4j.LoggerFactory
+import scala.util.Try
+import scala.util.Success
+import scala.util.Failure
+
+object TargetHost {
+
+  private[this] val logger = LoggerFactory.getLogger(this.getClass)
+
+  case class TargetHostException(msg: String) extends Exception(msg: String)
+
+  /**
+   * Reads from configuration file or environmental variable the Identity API
+   * host against which the tests should run. The host can be in LOCAL, CODE or
+   * PROD.
+   *
+   * Environmental variable IDENTITY_API_HOST has priority over
+   * identity.api.host property in application.conf file.
+   *
+   * TeamCity uses IDENTITY_API_HOST to target the tests.
+   *
+   * @return Identity API host
+   */
+  def idApiHost(): String =
+    readIdApiHostFromEnvVar orElse readIdApiHostFromConfigFile match {
+      case Failure(e) =>
+        logger.error("Target host not set. Specify target host either " +
+          "in environmental variable or application.conf")
+        sys.exit(1)
```

Comment:
Is this definitely the desired behaviour? It will exit the application on the first call to idApiHost() if not present, which may be after some tests have ran. If the purpose of the project is to run tests, shouldn't we use the test failure reporting mechanism rather than explicitly killing it?